dash-pipeline/SAI/sai_api_gen.py

```diff
@@ -403,6 +422,8 @@ def write_sai_files(sai_api):
                 for table_name in all_table_names:
                     if table_ref.endswith(table_name):
                         param[OBJECT_NAME_TAG] = table_name
+                        if is_table_deny_on_miss(sai_apis, table_name):
```

Comment:
I would prefer this logic not be buried in the API generator. Can we use a P4 annotation instead?